### DISCUSSION TITLE
Error: Operation failed due to concurrency.

### DISCUSSION THREAD

**Discussion by cuni0716** [alexandercerutti/passkit-generator#196] [Q&A]
Hello again @alexandercerutti!

I have an issue when I try to generate more than 1 pass at once:
```Error: Operation failed due to concurrency.```

Is there a way to enable concurrency somehow?

Thanks for the project and for your time answering!

**Comment by alexandercerutti** [comment]
Hi @cuni0716, that's completely new error to me as there is not concurrency mechanism in passkit-generator...

Could you please provide me a way to reproduce it?
If I don't know where this comes from, I won't be able to help you.

Let me know, thanks!

**Comment by cuni0716** [comment]
Sone @alexandercerutti, I've just created issue #197
